packages/data/src/world/oot/overworld.yml

```
@@ -1972,7 +1973,7 @@
     "Lake Hylia Fire Arrow": "is_adult && can_use_bow && (lake_water_control || scarecrow_longshot || longshot_anywhere || climb_anywhere)"
     "Lake Hylia HP": "can_ride_bean(BEAN_LAKE_HYLIA) || (is_adult && scarecrow_hookshot) || hookshot_anywhere"
     "Lake Hylia GS Lab Wall": "is_child && gs_night && (can_collect_distance || (trick(OOT_LAB_WALL_GS) && (can_use_sword || can_use_sticks)) || (time_travel_at_will && (scarecrow_hookshot || hookshot_anywhere || can_ride_bean(BEAN_LAKE_HYLIA)) && (has_explosives_or_hammer || can_use_din || can_use_slingshot)))"
-    "Lake Hylia GS Island": "is_child && gs_night && can_damage_skull"
+    "Lake Hylia GS Island": "is_child && gs_night && can_damage_skull && can_swim"
```

You can walk behind the island and longshot it from underwater.
```suggestion
    "Lake Hylia GS Island": "is_child && gs_night && ((can_damage_skull && can_swim) || (has_iron_boots && can_longshot))"
```